terraform/cloud-platform-components/resources/fluentd-es/templates/04-networkpolicy.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Please remove `NetworkPolicies` as well, we will be managing these in the environments repo.